libraries/AP_RCProtocol/AP_RCProtocol_DSM.cpp

```diff
@@ -250,11 +253,14 @@ bool AP_RCProtocol_DSM::dsm_parse_byte(uint32_t frame_time_us, uint8_t b, uint16
     if (byte_input.ofs >= AP_DSM_FRAME_SIZE) {
         start_frame_time_us = frame_time_us;
         byte_input.ofs = 0;
+        reset_rc_frame_count();
     }
 
     if (byte_input.ofs == 1) {
         // saw a beginning of frame marker
-        if (b == DSM2_1024_22MS || b == DSM2_2048_11MS || b == DSMX_2048_22MS || b == DSMX_2048_11MS) {
+        if (b == DSM2_1024_22MS || b == DSM2_2048_11MS ||
+            b == DSMX_2048_22MS || b == DSMX_2048_11MS ||
+            b == DSMX_2048_SAT || b == DSMX_2048_SAT2) {
```

Comment:
I don't understand these two. I was testing with a spectrum satellite. Where did they come from?

Reply:
If you look in https://www.spektrumrc.com/ProdInfo/Files/SPM4649T_Bi-Directional_SRXL_specifications.pdf at the bind types they are all listed consistently there. But 0x5B does not exist. What receiver do you have?

Reply:
I have a genuine spectrum satellite that came as a bundle with a AR9020 bought last year

Reply:
> at the bind types they are all listed consistently there.

except they are missing some :-)

Reply:
![image](https://user-images.githubusercontent.com/831867/90109785-a49c7400-dd8f-11ea-9fba-757bc24ece0f.png)

Reply:
Unfortunately I had a couple of these but destroyed both of them in crashes. My testing has been on a SPM4648 and a LemonRX. Both of these send the 0xb2 packet.
Are you binding with a genuine Spectrum TX?